### PR TITLE
[TNL-8180] - fix wiki validation error.

### PR DESCRIPTION
**Pull request by saadyousafarbi**
#### Fix for validation error for UnexpectedTag for `wiki` tag

### CODE REVIEW

**Review by awaisdar001 — CHANGES_REQUESTED** [review]
Please add some simple test

**Comment by saadyousafarbi** [comment]
@awaisdar001 test has been added. Named the new test course `testcourse13` as I already have [this](https://github.com/openedx/olxcleaner/pull/5) PR with `testcourse12` which I will merge first. 
It would be easier to rebase this way.

**Comment by jolyonb** [comment]
Note that you don't necessarily need to make a new test course every time you want to add in a new test. You can add the test piece to some of the courses that already exist (particularly for cases like this, where you're just adding a new tag).

**Comment by awaisdar001** [comment]
> Note that you don't necessarily need to make a new test course every time you want to add in a new test. You can add the test piece to some of the courses that already exist (particularly for cases like this, where you're just adding a new tag).

I think that makes sense @saadyousafarbi we can also tweak existing testcourses for cases that are possible.

File: testcourses/testcourse13/course/mycourseurl.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<course>
+  <wiki slug="test_wiki_slug"/>
```
Comment (awaisdar001) [inline testcourses/testcourse13/course/mycourseurl.xml:2]: Just add this piece of code in some other test course and we are good to go.

File: tests/test_validate.py [hunk 2]
```diff
@@ -161,3 +160,11 @@ def test_validate_course11():
         # Do not throw error when an xblock is allowed.
         course, errorstore, url_names = validate(**validate_kwargs, allowed_xblocks=current_allowed_xblocks)
         assert len(errorstore.errors) == total_errors - len(current_allowed_xblocks)
+
+def test_validate_course13():
```
Comment (awaisdar001) [inline tests/test_validate.py:164]: `test_wiki_xblock_is_supported`

File: tests/helpers.py [hunk 3]
```diff
@@ -12,6 +12,13 @@ def assert_error(errorstore, errorclass, file, msg):
             return
     raise ValueError("Error not found")
 
+def assert_not_error(errorstore, errorclass, file, msg):
+    """Search through the errorstore to assert that the given error is not present"""
+    for idx, error in enumerate(errorstore.errors):
+        if isinstance(error, errorclass) and error.filename == file and error.description == msg:
+            raise ValueError("Error not found")
```
Comment (jolyonb) [inline tests/helpers.py:19]: I think the error message is wrong?

**Review by awaisdar001 — APPROVED** [review]

**Pull request merged**